tests/solver-by-substitution.cc

```diff
@@ -846,3 +850,82 @@ BOOST_AUTO_TEST_CASE (rightHandSide)
         << error.norm() << ". Precision is " << test_precision);
   }
 }
+
+BOOST_AUTO_TEST_CASE (rightHandSideFromConfig)
+{
+  // Create a kinematic chain
+  DevicePtr_t device = hpp::pinocchio::unittest::makeDevice(
+      hpp::pinocchio::unittest::HumanoidSimple);
+  JointPtr_t ee1 = device->getJointByName ("lleg5_joint"),
+             ee2 = device->getJointByName ("rleg5_joint");
+  BOOST_REQUIRE (device);
+
+  ComparisonTypes_t comp1 (6, Equality);
+  comp1 [0] = comp1 [2] = comp1 [4] = EqualToZero;
+  ComparisonTypes_t comp2 (6, Equality);
+  comp2 [1] = comp2 [2] = EqualToZero;
+  // Create two relative transformation constraints
+  Transform3f tf1; // Identity
+  vector3_t u; u << 0, -.2, 0;
+  Transform3f tf2; tf2.translation (u);
+  ImplicitPtr_t c1 (hpp::constraints::implicit::RelativePose::create
+                    ("RelativeTransformation", device, ee1, ee2, tf1, tf2,
+                     std::vector <bool> (6, true), comp1));
+  u << 1.2, 0, -1;
+  tf2.translation (u);
+  ImplicitPtr_t c2 (hpp::constraints::explicit_::RelativePose::create
+                    ("Transformation", device, JointPtr_t (), ee1, tf2, tf1,
```

Comment:
I have a SEGV in this test. I think it comes from this line as the constructor arguments do not make sense to me. @florent-lamiraux Can you confirm this ?

Reply:
I precise that I am trying this with Ubuntu 18.04.

Reply:
The arguments seem correct to me. I will test in ubuntu-16.04.

Reply:
I don't understand how this function can be explicit. How does the function knows what SE3 joint values are the output ?

Reply:
You are right. It is not correct. It seems that we have never tested this in debug mode. I will fix the issue in a future pull request.